package.json

```
@@ -39,16 +39,17 @@
     "@docusaurus/core": "^2.0.1",
     "@docusaurus/preset-classic": "^2.0.1",
     "@emotion/cache": "^11.6.0",
-    "@emotion/react": "11.5.0",
+    "@emotion/react": "^11.5.0",
     "@emotion/server": "11.4.0",
-    "@emotion/styled": "11.3.0",
+    "@emotion/styled": "^11.3.0",
     "@golevelup/ts-jest": "^0.3.2",
     "@mdx-js/react": "^1.6.21",
     "@mui/icons-material": "^5.3.0",
     "@mui/lab": "^5.0.0-alpha.71",
-    "@mui/material": "^5.3.0",
+    "@mui/material": "^5.10.9",
     "@mui/styles": "^5.3.0",
     "@mui/system": "^5.3.0",
+    "@mui/x-data-grid": "^5.17.6",
```

I think we should try to minimize dependency changes (there's some pretty major changes in `package-lock`). Lets try to use the same version of data grid as the rest of our mui dependencies

```suggestion
    "@mui/x-data-grid": "^5.3.0",
```